src/GenesisTestsFiller/dummy.json

```diff
@@ -33,6 +31,7 @@
             "byzantiumForkBlock": "0x00",
             "chainID": "0x01",
             "constantinopleForkBlock": "0x00",
+            "daoForkBlock": "0x00",
```

Comment:
if this fields are not required, what is the default value? thats important cause some client has it set to 0x00 and other to infinity. 

also the names of the fields are important issue here. Parity has EIP155. 
are devs agreed to use homesteadForkBlock, byzantiumForkBlock, EIP158ForkBlock for the fork names?